quipucords/api/common/entities.py

```diff
@@ -148,7 +147,7 @@ def ipv6_addresses(self):
     def mac_addresses(self):
         """Retrieve mac_addresses."""
         with suppress(TypeError):
-            return json.loads(self._fingerprints.mac_addresses)
+            return self._fingerprints.mac_addresses
```

Comment:
These properties normalize the access from the _fingerprints relation (SystemFingerprint), so I'd still keep it, but we can remove the suppress TypeError, same with the ip_addresses property.

Reply:
mac_addresses and ip_addresses properties updated.

Reply:
FYI `HostEntity` has some dark magic in its `__gettattr__` implementation. Every time an attribute is not found in `HostEntity` we look up `_fingerprints`

(no need to change, tho. explicit is better anyways)

https://github.com/quipucords/quipucords/blob/a25cae869dd71e0287d17bf94e30203e823268a4/quipucords/api/common/entities.py#L46-L55

Reply:
Uh oh, dark magic.  yeah, I guess as long as the uses don't get too verbose (if ... get from here otherwise get from there) in too many places, we can do a follow-up PR